jobbergate-api/pyproject.toml

```diff
@@ -28,21 +28,21 @@ python-multipart = "^0.0.5"
 python-dotenv = "^0.20.0"
 boto3 = "^1.17.51"
 Jinja2 = "^3.0.3"
-asyncpg = "^0.22.0"
+asyncpg = "^0.27"
 loguru = "^0.6.0"
 uvicorn = "^0.15.0"
 armasec = "^0.11.0"
 sentry-sdk = "^1.5.0"
 yarl = "^1.7.2"
 alembic = "^1.7.5"
 typer = "^0.4.0"
-databases = {extras = ["postgresql"], version = "^0.5.5"}
```

Comment:
I know we've decided to replace databases with sqlalchemy on Jobbergate 4.x, but for the multi-tenancy feature, was databases a particular problem?

Reply:
Given the way we have to create engines on demand and make an async session for each request, it seemed like it would be easier to just make the jump. Since we couldn't use the global `database` module any more, there really wan't any utility left for the `databases` package.

Reply:
Yeah, thanks for that, the refactor to sqlalchemy looks great